src/Element/Element.php

```diff
@@ -82,16 +82,6 @@ protected function getDriver()
     }
 
     /**
-     * Returns selectors handler.
-     *
-     * @return SelectorsHandler
-     */
-    protected function getSelectorsHandler()
```

Comment:
I guess I need to do `Escaper::escapeLiteral` when I need to escape xpath instead of doing 

```php
$selectors_handler = $this->getSelectorsHandler();
$selectors_handler->xpathLiteral($text)
```

Was this `Escaper` class part of Mink 1.6 release?